### ISSUE TITLE
On macOS, the location of sclang should be moved to `SuperCollider.app/Contents/Resources/'

### ISSUE THREAD

**Issue by prko** [supercollider/supercollider#6204]
## Environment

* SuperCollider version: 3.13.0 and 3.14.0_dev (probably all SC with SC-IDE on MacOS only).
* Operating system: macOS 13.6.4.

## Steps to reproduce

1. run `sclang` in `/Applications/SuperCollider.app/Contents/MacOS` inside vscode or in a terminal window using a port number that is not the default port.
2. run `/Applications/SuperCollider.app/`.
3. `/Applications/SuperCollider.app/` does not start. 

## Expected vs. actual behaviour
- Expected: `sclang` (`/Applications/SuperCollider.app/Contents/MacOS/sclang`) and `scied` (`/Applications/SuperCollider.app/Contents/MacOS/SuperCollider`) start separately.
- Actual: `scide` (`/Applications/SuperCollider.app/Contents/MacOS/SuperCollider`) will not start if (`/Applications/SuperCollider.app/Contents/MacOS/sclang`) is already running.

## How to fix it?
I think the location of `sclang` should be moved to `/Applications/SuperCollider.app/Contents/Resources/'.

## Reference
https://scsynth.org/t/can-sclang-detect-the-ide-it-is-currently-using/8873/7?u=prko

**Comment by mossheim** [comment]
This is not a supported use case.

**Comment by prko** [comment]
If sclang is frozen, this behaviour is an obstacle to finding sclang because sclang and SC-IDE are shown as SuperCollider with the same icon in the Force Quit Application window.

**Comment by muellmusik** [comment]
But only one will show as ‘not responding’, or?

**Comment by prko** [comment]
Yes, but I should wait to see one not responding for at least several seconds or very longer than 10 soconds (sometimes 20 secs, I feel).

Reactions are not instantaneous.

**Comment by dyfer** [comment]
Let's try to use issue description as the title, instead of a proposed solution which might not be valid.

**Comment by dyfer** [comment]
I was mostly referring to the issue _title_ - I renamed this issue, since the title wasn't describing the issue itself.
